src/session-pool.ts

```diff
@@ -80,15 +85,23 @@ export interface SessionPoolInterface extends EventEmitter {
    *
    * @name SessionPoolInterface#open
    */
+  /**
+   * When called returns a session.
+   *
+   * @name SessionPoolInterface#getSession
+   * @param {GetSessionCallback} callback The callback function.
+   */
   /**
    * When called returns a read-only session.
```

Comment:
nit: I think we should update this part of the documentation as well to indicate that it always just returns a session. The same also applies to the GetWriteSessionCallback below.

Reply:
done